### PR TITLE
Wells/auth handler

### PR DESCRIPTION
**Pull request by wellsjo**
- send subscriptions and version with auth message
- don't send identification message

### CODE REVIEW

File: stream_test.go [hunk 1]
```diff
@@ -1050,49 +1025,6 @@ func waitConnOpen(t *testing.T, tp *testServerParams) error {
...
-		// Check useragent
-		{
-			want := "Cryptowatch Stream Client Golang"
-			got := strings.Split(cid.Useragent, "/")[0]
-			if got != want {
-				return errors.Errorf("Useragent: want: %q, got: %q", want, got)
-			}
-		}
-
-		// Check subscriptions
```
Comment (dimonomid) [inline stream_test.go]: Then how about adding the check for subs (and probably version) to `waitAuthnReq`?

Comment (wellsjo) [reply stream_test.go]: i was working on this  :)

Comment (dimonomid) [reply stream_test.go]: :) :+1:

Comment (wellsjo) [reply stream_test.go]: done

**Pull request merged**